src/test/resources/application.yaml

```diff
@@ -1,15 +1,31 @@
 uv:
-  security:
-    auth: development
   calendar:
     organizer: organizer@example.org
   mail:
     application-url: https://localhost:8080
     sender: sender@example.org
     administrator: administrators@example.org
     senderDisplayName: Urlaubsverwaltung
+  security:
+    oidc:
+      login-form-url: http://localhost:8080/login/redirect
 
 spring:
+  security:
+    oauth2:
+      client:
+        registration:
+          default:
+            client-id: urlaubsverwaltung
+            provider: default
+            scope: openid,profile,email,roles
+            authorization-grant-type: authorization_code
+            redirect-uri: http://{baseHost}{basePort}/login/oauth2/code/{registrationId}
+        provider:
+          default:
+            token-uri: https://auth.example.org/realms/2657fdc6/protocol/openid-connect/token
```

Comment:
2657fdc6 -> urlaubsverwaltung-realm

Reply:
yeah, but it does not matter. there has be be "anything"